UBUNTU.md

```diff
@@ -285,6 +285,14 @@ exit
 rm -f /tmp/caller
 ```
 
+## Tracked directories
+
+Ubuntu system only tracks a certain number of directories. This limit is very low by default. To raise this limit, run the following line in your terminal:
+
+```bash
+echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
+```
+If you encounter an error, please call a teacher.
```

Comment:
I think this last line ("If you encounter [....]") isn't really necessary, they already do that :)